Resources/config/sitemap.xml

```diff
@@ -29,31 +29,31 @@
 
         <service id="cmf_seo.sitemap.guesser_chain" class="%cmf_seo.sitemap.guesser_chain.class%" />
 
-        <service id="cmf_seo.sitemap.guesser.seo_metadata_title" class="%cmf_seo.sitemap.guesser.seo_metadata_title.class%" >
+        <service id="cmf_seo.sitemap.guesser.seo_metadata_title" class="%cmf_seo.sitemap.guesser.seo_metadata_title.class%" public="false">
             <argument type="service" id="cmf_seo.presentation" />
-            <tag name="cmf_seo.sitemap.guesser" />
+            <tag name="cmf_seo.sitemap.guesser" priority="-2"/>
```

Comment:
Isn't -2 smaler than 0? And wouldn't that let the users guesser come after the provided ones?